src/Grids/rectilinear_grid.jl

```diff
@@ -518,19 +518,12 @@ const C = Center
 @inline ηnodes(grid::RG, ℓx, ℓy, ℓz; kwargs...) = ynodes(grid, ℓy; kwargs...)
 @inline rnodes(grid::RG, ℓx, ℓy, ℓz; kwargs...) = znodes(grid, ℓz; kwargs...)
 
+@inline isrectilinear(::RG) = true
+
 #####
-##### Grid spacings
+##### Grid-specific grid spacings
 #####
 
-@inline xspacings(grid::RG, ℓx::C; with_halos=false) = _property(grid.Δxᶜᵃᵃ, ℓx, topology(grid, 1), size(grid, 1), with_halos)
-@inline xspacings(grid::RG, ℓx::F; with_halos=false) = _property(grid.Δxᶠᵃᵃ, ℓx, topology(grid, 1), size(grid, 1), with_halos)
-@inline yspacings(grid::RG, ℓy::C; with_halos=false) = _property(grid.Δyᵃᶜᵃ, ℓy, topology(grid, 2), size(grid, 2), with_halos)
-@inline yspacings(grid::RG, ℓy::F; with_halos=false) = _property(grid.Δyᵃᶠᵃ, ℓy, topology(grid, 2), size(grid, 2), with_halos)
-@inline zspacings(grid::RG, ℓz::C; with_halos=false) = _property(grid.Δzᵃᵃᶜ, ℓz, topology(grid, 3), size(grid, 3), with_halos)
-@inline zspacings(grid::RG, ℓz::F; with_halos=false) = _property(grid.Δzᵃᵃᶠ, ℓz, topology(grid, 3), size(grid, 3), with_halos)
-
-@inline xspacings(grid::RG, ℓx, ℓy, ℓz; kwargs...) = xspacings(grid, ℓx; kwargs...)
-@inline yspacings(grid::RG, ℓx, ℓy, ℓz; kwargs...) = yspacings(grid, ℓy; kwargs...)
-@inline zspacings(grid::RG, ℓx, ℓy, ℓz; kwargs...) = zspacings(grid, ℓz; kwargs...)
-
-@inline isrectilinear(::RG) = true
+@inline xspacings(grid::RG, ℓx) = xspacings(grid, ℓx, Center(), Center())
+@inline yspacings(grid::RG, ℓy) = yspacings(grid, Center(), ℓy, Center())
+@inline zspacings(grid::RG, ℓz) = zspacings(grid, Center(), Center(), ℓz)
```

Comment:
```suggestion
@inline xspacings(grid::RG, ℓx) = xspacings(grid, ℓx, nothing, nothing)
@inline yspacings(grid::RG, ℓy) = yspacings(grid, nothing, ℓy, nothing)
@inline zspacings(grid::RG, ℓz) = zspacings(grid, nothing, nothing, ℓz)
```

Reply:
Note that it is incorrect to denote the locations in the other directions as `Center()` (or in other words the location could just as well be denoted `Face()`). I think `nothing` is correct here.

Reply:
I take this as a sign that my evil plan is working 😈